bootstrap.sh

```diff
@@ -15,6 +15,8 @@ kubectl apply -f .infrastructure/app/nodeport.yml
 kubectl apply -f .infrastructure/app/hpa.yml
 kubectl apply -f .infrastructure/app/deployment.yml
 
+kubectl apply -f .infrastructure/security/rbac.yml
+
 # Install Ingress Controller
 kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
 # kubectl apply -f .infrastructure/ingress/ingress.yml
```

Comment:
The ingress configuration is commented out. If ingress is required for your application, make sure to uncomment this line and verify that the `ingress.yml` file is correctly set up.